### DISCUSSION TITLE
Upcoming third-party SDK requirements

### DISCUSSION THREAD

**Discussion by allywuyi** [SDWebImage/SDWebImage#3651] [Q&A] [answered]
Hello, the question I would like to inquire about concerns Apple's new privacy requirements: [Upcoming third-party SDK requirements](https://developer.apple.com/support/third-party-SDK-requirements/).

I have installed the latest version of SDWebImage (5.18.7) through CocoaPods and have observed the presence of PrivacyInfo.xcprivacy. Additionally, I would like to ask about the "Signatures for SDKs" mentioned in the above link. Is it required for SDWebImage, or does this latest version of the SDK already include signatures?

Thank you in advance!

**Comment by dreampiggy** [comment, marked as answer]
> I would like to ask about the "Signatures for SDKs" mentioned in the above link

It's about the binary distribution, which means, a `xcframework` or `framework` binary can be modified by the third-party people (not me)

We do not provide the original binary destribution, since if you use SPM or CocoaPods, it will use Xcode to build from source code, and do the correct codesign. So it should just work
